<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventEditorDialog</class>
 <widget class="QDialog" name="EventEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Событие</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/icons/event_icon</normaloff>:/new/icons/event_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Название события:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="text">
        <string>Безымянное событие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Тип события:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Город события:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cityComboBox">
       <item>
        <property name="text">
         <string>Москва</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Нью-йорк</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алматы</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Владимир</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Воронеж</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Днепропетровск</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дубна</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Екатеринбург</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Запорожье</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленогорск</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Казань</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Киев</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Минск</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Одесса</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рига</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Санкт-Петербург</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тюмень</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Харьков</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Дата события:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EventEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EventEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
